components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/add-wf-wizard.jsp

```diff
@@ -89,6 +91,13 @@
             }
             session.setAttribute(requestToken,workflowWizard);
         }
+
+
+        WorkflowWizard[] workflowWizards = client.listWorkflows();
```

Comment:
Isn't it good to use ArrayList<WorkflowWizard> here, rather than a static array[]?

Reply:
fixed